docs/paymentMethods/paymentMethodGroups.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Payment Method Groups (PMG)
+sidebar_label: Payment Method Groups (PMG)
+description: Learn about Payment Method Groups
+keywords:
+  - payment methods
+  - payment method groups
+  - pmg
+---
+
+# Payment Method Groups (PMG)
+
+Payment Method Groups offer a way to filter payment methods by a group.
```

Comment:
Please update to:
"Payment method groups offer a way to organize and filter payment methods of your users. 
It also allows you to display already save payment methods to a user in the checkout form when using [Hosted Checkout]() or one of the checkout web components ([Unified Fintech Checkout]() or [Modular Checkout]()). 
Here is how to use it:
1. Create a payment method group
2. Associate existing payment methods
3. Create a checkout and pass the payment method group ID